perfact/zodbsync/zodbsync.py

```diff
@@ -461,7 +463,9 @@ def fs_read(self, path, encoding=None):
             # Translate file system data
             meta = dict(fix_encoding(meta, encoding))
 
-        return meta
+        # to compare zodb and fs object meta we need to decode them the same
+        # way
+        return meta, meta_str.decode('utf-8')
```

Comment:
Also, in Python 2 this will return `unicode` while the comparison later is against `str`. This might break if there are non-ascii characters. Please use `to_string` instead.